classes/05class/exercises/c05-make01/deniseddi/output.md

```diff
@@ -0,0 +1,39 @@
+# Output of command "make build"
+
+\$ make build
+echo "This image was created by Denise based on the 4e24982" > index.html
+docker build -t make01:4e24982 .
+Sending build context to Docker daemon 4.096kB
+Step 1/2 : FROM httpd:2.4.41-alpine
+2.4.41-alpine: Pulling from library/httpd
+aad63a933944: Pull complete
+29ade582b51e: Pull complete
+7e41ad5b6f9c: Pull complete
+ebf61b47b4ca: Pull complete
+9c060bce4eae: Pull complete
+Digest: sha256:06ad90574c3a152ca91ba9417bb7a8f8b5757b44d232be12037d877e9f8f68ed
+Status: Downloaded newer image for httpd:2.4.41-alpine
+---> 54b0995a6305
+Step 2/2 : COPY index.html /usr/local/apache2/htdocs/
+---> 24e3de09c4de
+Successfully built 24e3de09c4de
+Successfully tagged make01:4e24982d
+
+# Output of command "make push"
+
+\$ make push
+docker tag make01:4e24982 portoddi/make01:4e24982
+docker push portoddi/make01:4e24982
+The push refers to repository [docker.io/portoddi/make01]
+4bdcd90120cd: Pushed
+8e24b43b535f: Pushed
+1dfb4a5d0ecd: Pushed
+3313c0c04ade: Pushed
+310889822143: Pushed
+beee9f30bc1f: Pushed
+4e24982: digest: sha256:564a70c9997d173727d55a72d5ed17c0a9675706224c59ea94c0998377a7ba7c size: 1569
+
+#Output of command "make run"
+\$ make run
+docker run -p 80 -d portoddi/make01:4e24982
+203202a88413f720660dc4034717660a5919386b980f5ae307bc88e061c6887b
```

Comment:
Can you include the logs without detaching the container so I can actually see the output?

Reply:
Hi Denis!
I am reviewing this exercise.
I left the "Expose 80" uncommented :) 

Now, without the detach command `denisegomes$ make run
docker run -p 80 portoddi/make01:dd2d4f0` I get this output: (Is this what you expected?) 

docker run -p 80 portoddi/make01:dd2d4f0
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
[Mon May 11 06:20:13.006856 2020] [mpm_event:notice] [pid 1:tid 139903113719112] AH00489: Apache/2.4.41 (Unix) configured -- resuming normal operations
[Mon May 11 06:20:13.009952 2020] [core:notice] [pid 1:tid 139903113719112] AH00094: Command line: 'httpd -D FOREGROUND'
[Mon May 11 06:20:26.306513 2020] [mpm_event:notice] [pid 1:tid 139903113719112] AH00492: caught SIGWINCH, shutting down gracefully

Reply:
That sounds like what Denis asked for 😄 !

Reply:
Glad to see you commenting here @starkmatt !
Feel free to jump in on PRs of the new students too, comment and approve them 🤝 

Thanks @deniseddi